<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivationDialog</class>
 <widget class="QDialog" name="ActivationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activation</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QCheckBox" name="checkBoxLicenseFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LicenseFileでActivate</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxActivationCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OfflineでActivate</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOrderCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ライセンスキーでActivate</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameOrderCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonActivateByOrderCode">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;メールで受信したライセンスキーを貼り付け、&lt;/p&gt;&lt;p&gt;インターネットに接続してください&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOrderCode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameLicenseFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;メールで受信したLicenseFileを選択し、、&lt;/p&gt;&lt;p&gt;インターネットに接続してください&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonActivateByLicenseFile">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectLicenseFiile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLicenseFileName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameActivationCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonLoadLicenseFiile">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load License file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1:License fileを読込&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2: 認証要請ファイルを出力してMEGATRADEに送信&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonWriteReqActivationFile">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Write ReqActivation file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3:認証ファイルを読込</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadActivationFile">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Activation file</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLicenseKey">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>201</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
